CIFAR10/train/KRes34_1GTX1080Ti_pytorch.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Chen Wang",
+    "authorEmail": "wang.chen@zoho.com",
+    "framework": "PyTorch",
```

Comment:
What version of PyTorch did you use? Please at the version number or git commit hash.